src/odoc/bin/main.ml

```diff
@@ -377,6 +377,10 @@ module Odoc_html = Make_renderer (struct
     in
     Arg.(value & flag (info ~doc [ "closed-details" ]))
 
+  let indent =
+    let doc = "Format the output HTML files with indentation" in
+    Arg.(value & flag (info ~doc [ "indent" ]))
```

Comment:
What about enabling it by default without exposing the option ? Whitespaces compress well and indented output is always nicer to look at.

Reply:
could do - I've got no strong views on this. I've added it because my previous script was just running things through `html tidy`, and the larger my test set got the slower the tidying got!